app/src/main/java/com/sunasterisk/a14day_challenge/ui/SplashActivity.kt

```diff
@@ -9,7 +9,7 @@ class SplashActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        startActivity(LoginActivity.getIntent(this))
+        startActivity(LoginActivity.getLoginScreenIntent(this))
```

Comment:
```suggestion
        startActivity(LoginActivity.getIntent(this))
```
chỉ cần getIntent là được, k thì sẽ lặp LoginScreen